src/core/logging.py

```
@@ -0,0 +1,215 @@
+"""
+Logging configuration for the Fraud Detection System.
+
+Provides structured logging with file rotation and multiple output formats.
+"""
+
+import logging
+import logging.handlers
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)